Shared/mods/deathmatch/logic/CScriptDebugging.cpp

```diff
@@ -55,6 +55,21 @@ void CScriptDebugging::LogCustom(lua_State* luaVM, unsigned char ucRed, unsigned
     VSNPRINTF(szBuffer, MAX_STRING_LENGTH, szFormat, marker);
     va_end(marker);
 
+    SLuaDebugInfo luaDebugInfo;
+    LogString("", luaDebugInfo, szBuffer, 0, ucRed, ucGreen, ucBlue);
```

Comment:
The new LogCustom still uses 0 as the uiMinimumDebugLevel so that we don't have to add a 4th level to /debugscript and instead simply show them in the same level as previous custom-level messages. The differenciation between LogCustom and LogDebug is based on whether `DEBUG_INFO_NONE` of SLuaDebugInfo is used instead of real debug info.